apps/nextra/pages/en/build/smart-contracts/why-move.mdx

```diff
@@ -4,90 +4,144 @@ title: "Why Move?"
+coverage reporting.
+  - **Decompiler**: For better security, on-chain bytecode can be disassembled or
+decompiled to provide visibility into the actual contracts.
+  - **IDE Plug-ins**: Aptos has support for two major IDEs:  [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=MoveBit.aptos-move-analyzer) and [IntelliJ](https://plugins.jetbrains.com/plugin/14721-move-on-aptos).
 
-2. **IDE Plug-ins**: Aptos has language plug-ins for top two IDEs: [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=MoveBit.aptos-move-analyzer), [Intellij](https://plugins.jetbrains.com/plugin/14721-move-on-aptos).
+2. **Data Model**: Aptos has an accessible data model with the data definition
+stored on-chain. Objects and accounts can have multiple distinct structures in
+an easy-to-parse format.
 
-3. **Data Model**: Aptos has a more accessible data model with the data definition stored on-chain and objects and accounts can have multiple distinct structures in an easy-to-parse format.
+3. **Upgradability**: Upgradability ensures that application interfaces cannot
+be broken and doesn't require explicit adoption from downstream applications.
+Contracts can simply be upgraded in-place to fix bugs.
 
-4. **Safe Upgradeability**: Unique safe upgradeability ensures that application interfaces cannot be broken and doesn't require explicit adoption from downstream applications.
+4. **Cross-Interaction**: Move allows for interaction between contracts by using
+type-safe structs.
 
-5. **Cross-Interaction**: Move allows for more intuitive cross (module, contract, program) interaction by using type-safe structs.
+5. **Code Storage**: Aptos stores source code on-chain improving the ability to
+audit and ensure contract to bytecode correctness.
 
-6. **Code Storage**: Aptos stores code on-chain improving the ability to audit and ensure contract to byte code correctness.
+6. **Sponsored Transactions**: Native sponsored transaction support allows for
+having transactions be paid by other users with no special services or
+contract-specific code required.
 
-7. **Sponsored Transactions**: Only application with native sponsored transaction support — no special services or contract-specific code required.
+7.  **Robust Token Standards**: The Digital Asset and Fungible Asset standards
+provide flexibility and a unified standard for diverse types of tokens and
+digital assets on-chain.  These were influenced by existing standards such as
+ERC-20, ERC-721, ERC-1155 and Token-2022.
 
-8. **Token Types**: Like [Token-2022](https://spl.solana.com/token-2022), Fungible Asset and Digital Asset cover the basis in what is possible within the token space without introducing additional token types.
+8. **On-chain Randomness**: Native on-chain unbiasable randomness provides a safe
+and consistent way of getting random numbers, with extra safety checks at compile
+time.
 
 
 ## Comparison to Other Blockchains
 
```

Comment:
I thinks its fine right now, but do not promise anything about not having dynamic dispatch (no re-entrency is fine though)